governance/votes/Executive vote - May 18, 2022.md

```diff
@@ -0,0 +1,78 @@
+- 541.47 MKR will be distributed to the [Sustainable Ecosystem Scaling Core Unit (SES-001)](https://mips.makerdao.com/mips/details/MIP39c2SP10) as detailed below.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- (Foundational L2) A new `DssCure` module will be added and authorized.
+- (Foundational L2) `MCD_END` (responsible for Emergency Shutdown logic) will be replaced.
+- (Foundational L2) The existing ESM contract will point to the new `END`.
+- A new Flash Mint Module will be added and authorized.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit MKR Vesting Streams
+
+As per their successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17), a total of **541.47 MKR** will be distributed to the [Sustainable Ecosystem Scaling Core Unit (SES-001)](https://mips.makerdao.com/mips/details/MIP39c2SP10) at [0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6](http://etherscan.io/address/0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6) (SES-001's Auditor Wallet) if this executive proposal passes.
+
+### Foundational L2 Work
+
+The [Maker Teleport](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) initiative, [previously known as Maker Wormhole](https://forum.makerdao.com/t/maker-wormhole-new-name-maker-teleport/15115), will be bringing canonical DAI capability onto L2 domains. As part of this development, it is necessary to introduce a new module called `DssCure` and update the existing `End` module to interact with it.
+
+In the event of an [Emergency Shutdown](https://docs.makerdao.com/smart-contract-modules/shutdown), the `DssCure` module ensures a fair distribution to DAI holders by calculating how much debt needs to be reduced where there is pre-minted, unused DAI sitting in the `Vat`. Discounting these pre-minted, out-of-circulation DAI is necessary for the correct calculation of the outstanding DAI---otherwise, the artificial, increased debt resulting from factoring in the pre-minted DAI will result in DAI holders receiving less collateral than they are entitled to.
+
+`DssCure` sits between the `Vat` and `End` so that `End` receives the correct amount of debt.
+
+The following additions and changes will take place if this executive proposal passes:
+- A new `DssCure` module will be added to the Maker Protocol.
```

Comment:
You might want to call it `MCD_CURE` referring to its future chainlog key.

Reply:
Kept DssCure references, but clarified it will be added to the chainlog as MCD_CURE.